packages/flutter_localizations/lib/src/l10n/generated_date_localizations.dart

```diff
@@ -21962,12 +22668,12 @@ const Map<String, Map<String, String>> datePatterns =
     'H': 'H时',
     'Hm': 'HH:mm',
     'Hms': 'HH:mm:ss',
-    'j': 'ah时',
-    'jm': 'ah:mm',
-    'jms': 'ah:mm:ss',
-    'jmv': 'v ah:mm',
-    'jmz': 'z ah:mm',
-    'jz': 'zah时',
+    'j': 'H时',
```

Comment:
Why remove the pattern `a`, it will caused the localization `AMPMS`, Does it means need to adaptor the origin logical by append an extra patterns for it? such as 
```
    if (!alwaysUse24HourFormatValue) {
      DateFormat.yMd('zh').addPattern('a', '').add_jm().format(DateTime.now())
    }
```